### ISSUE TITLE
Make links clickable in student news

### ISSUE THREAD

**Issue by Matt-Ramos** [gordon-cs/gordon-360-ui#881]
when others share a link in student news (often to register for events) and it is clicked on, the student news item component collapses.

**Comment by andrew-wzj** [comment]
I wonder just make the link on the news post to be hyperlink, and how it collapses ( not responding or quit the news tab)

**Comment by andrew-wzj** [comment]
The real question is should we want the link to work (concern about security on redirecting website)

**Comment by bennettforkner** [comment]
I agree with Matt. We still require that all student news postings be approved by Student Life, so I wouldn't have an issue security-wise.
